src/testing/flow_solver_cases/flow_solver_case_base.cpp

```diff
@@ -26,6 +26,14 @@ void FlowSolverCaseBase<dim,nstate>::display_flow_solver_setup(std::shared_ptr<I
     pcout << "- PDE Type: " << pde_string << std::endl;
     pcout << "- Polynomial degree: " << this->all_param.grid_refinement_study_param.poly_degree << std::endl;
     pcout << "- Final time: " << this->all_param.flow_solver_param.final_time << std::endl;
+
+    this->display_additional_flow_case_specific_parameters(initial_condition);
```

Comment:
Why is `this` keyword being used in the base class, and why is it calling a function that does nothing?

Reply:
Personal preference, I like it since using `this` explicitly marks the object members

Reply:
For `display_additional_flow_case_specific_parameters`, see comment https://github.com/dougshidong/PHiLiP/pull/139#discussion_r867372889